markymark/Classes/Default implementations/MarkDownTextView.swift

```diff
@@ -90,6 +90,14 @@ open class MarkDownTextView: UIView {
         markyMark.addRule(rule)
     }
 
+    public func addViewLayoutBlockBuilder(layoutBlockBuilder: LayoutBlockBuilder<UIView>) {
```

Comment:
We could use `_` as argument label to shorten the call site. The repetition of `layoutBlockBuilder` is too verbose for my taste. Same for method below.